src/js/oareport.js

```diff
@@ -483,7 +483,7 @@ oareport = function(org) {
       xhr.open("GET", query);
       // Display message when server responds
       xhr.onload = function () {
-        document.querySelector("#csv_email_msg").innerHTML = "OA.Report has started building your CSV export at <a href='" + this.response + "' target='_blank' class='underline'>this URL</a>. Please check your email to get the full data once it’s ready.";
+        document.querySelector("#csv_email_msg").innerHTML = "OA.Report has started building your CSV export at <a href='" + this.response + "' target='_blank' class='underline' id='exportLink'>this URL</a>. Please check your email to get the full data once it’s ready.";
```

Comment:
@JosephMcArthur What is this change for? Also, it doesn’t use a consistent naming convention with the rest of the code

Reply:
See: https://github.com/oaworks/Gates/issues/502#issuecomment-1511083002. Supports getting the link more easily for the test.